main.cpp

```diff
@@ -50,10 +50,18 @@
 #include "daemon/DaemonManager.h"
 #include "AddressBook.h"
 #include "model/AddressBookModel.h"
+#include "wallet/wallet2_api.h"
 
+void messageHandler(QtMsgType type, const QMessageLogContext &context, const QString &msg)
+{
+    // Send all message types to logger
+    Monero::Wallet::debug(msg.toStdString());
+}
```

Comment:
I would add another parameter here, the category. Otherwise all of these will end up with the generic category defined in src/wallet/api/wallet.cpp (maybe wallet or default). But maybe that's enough ? If you think it is, I'm fine with it.